### DISCUSSION TITLE
Moving around a model not working in 0.17 + Text labels?

### DISCUSSION THREAD

**Discussion by Yageller** [isl-org/Open3D#6524] [General]
Hi! I am trying to render a 3D mesh, render looking at it from a certain angle, add some text labels in 3D space. 
Sounds simple...? didn't turn out so.
- I first tried on version 0.17. Turns out that all of my attempts to use vis.get_view_control() to move around the model have no effect. I racked my brain (and the internet) for hours. Turns out it simply doesn't work in 0.17 (things like change_field_of_view. set_front, set_lookat etc...). when I downgraded to 0.16 - all worked as expected...
- I then tried to add 3D Labels for text. unfortuantelly, this only works in the gui / Application interface with O3DVisualizer and not the regular Visualizer... if I try to use O3DVisualizer (which seems to be deprecated in 0.17) I can add 3d labels, but I loose the ability to move about using  vis.get_view_control()...

so - seems like lots of changes are happening. Questions:
- I assume that methods in vis.get_view_control() like change_field_of_view. set_front, set_lookat not working in 0.17 is a bug?
- Is there any way to add text labels to the regular Visualizer?

Thanks!

**Comment by saurabheights** [comment]
@Yageller Partial answer - Visualizer bug of get_view_control and other methods not working is fixed in master. If you install development release, that should resolve your issue. However, `Visualizer` does not support 3d Label.
